src/classes/RD2_OpportunityEvaluationService_TEST.cls

```diff
@@ -238,17 +245,212 @@ private class RD2_OpportunityEvaluationService_TEST {
         System.assertEquals(false, oppById.get(oldOpps[1].Id).isClosed, 'Open Opp should stay open');
     }
 
+    /**
+    * @description When Installment Auto Creation Setting is set to Disable First Installment, 
+    * the installment should not be created on insert of RD
+    */
+    @IsTest
+    private static void shouldNotCreateOppOnRDInsertWhenDisableFirstInstallment() {
+        RD2_EnablementService_TEST.setRecurringDonations2Enabled();
+
+        Map<String, Object> rdSettingMap = new Map<String, Object> {
+            'InstallmentOppAutoCreateOption__c' => OPTION_NAME_DISABLE_FIRST
+        };
+
+        RD2_Settings_TEST.setUpConfiguration(rdSettingMap);
```

Comment:
To minimize code review back and forth with regards to naming as well as to reduce number of line, the `rdSettingMap ` can be removed and the next statement can be:
```
       RD2_Settings_TEST.setUpConfiguration(new Map<String, Object> {
            'InstallmentOppAutoCreateOption__c' => OPTION_NAME_DISABLE_FIRST
        });
```